pkg/cmd/server/middleware.go

```diff
@@ -210,3 +214,225 @@ func (mc *MiddlewareChain[T]) modify(modifications ...MiddlewareModification[T])
 	}
 	return nil
 }
+
+var none = uuid.NewString()
+
+func inTest() bool {
+	return flag.Lookup("test.v") != nil
```

Comment:
can we use the function we already have defined in the testutil package?

Reply:
I couldn't find any. Could you point me to it?

Reply:
https://github.com/authzed/spicedb/blob/5d33a67c26f89275f839ee7841557c3552c45cd9/pkg/spiceerrors/bug.go#L10

Reply:
it's unexported. If you are cool with that, I can export it